packages/contracts/src/manager/interfaces/IDiamondCut.sol

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IDiamondCut {
```

Comment:
According to the spec this should be implemented within IDiamond and not IDiamondCut

Reply:
> According to the spec this should be implemented within IDiamond and not IDiamondCut

Yeah, it can be, but don't have to do it